<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>driverPanel</class>
 <widget class="QWidget" name="driverPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="4">
      <widget class="QPushButton" name="endTripCmd">
       <property name="text">
        <string>&amp;End trip</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="arrivedCmd">
       <property name="text">
        <string>&amp;Arrived</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="logoutCmd">
       <property name="text">
        <string>Log&amp;out</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="statusTab">
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="1">
        <widget class="QLabel" name="locationLbl">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QTreeWidget" name="universeTree">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="availableCheck">
         <property name="text">
          <string>Available</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tripTab">
      <attribute name="title">
       <string>Trips</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="requestsTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="reloadCmd">
           <property name="text">
            <string>Re&amp;load Requests</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>188</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="acceptCmd">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>A&amp;ccept</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>endTripCmd</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>universeTree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
